src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -267,6 +269,23 @@ export class StudyViewPageStore {
 
     @observable private chartsType = observable.map<ChartType>();
 
+    @autobind
+    isChartHighlighted(uniqueKey: string): boolean {
+        // Highlight survival analysis chart
+        if (this.analysisGroupsSettings.clinicalAttribute &&
+            (getClinicalAttributeUniqueKey(this.analysisGroupsSettings.clinicalAttribute) === uniqueKey)) {
+            return true;
+        }
+        const date = this._chartToBeHighlighted.get(uniqueKey);
+        if (date === undefined) {
+            return false;
+        } else if (new Date().getTime() - date.getTime() < 5000) {
```

Comment:
I think this time sensitive condition deserves a comment why we are doing it this way